cache/client.go

```diff
@@ -17,6 +17,7 @@ import (
 // Common functionality shared between the Memcached and Redis Cache implementations
 
 const (
+	opAdd            = "add"
```

Comment:
this isn't used to initialize `cm.operations`

Reply:
Fixed